_sips/minutes/2017-05-08-sip-minutes.md

```diff
@@ -0,0 +1,93 @@
+
+### [SIP-NN - comonadic-comprehensions](http://docs.scala-lang.org/sips/comonadic-comprehensions.html)
+[YouTube time: 1:39](https://youtu.be/6rKa4OV7GfM?t=99)
+
+Proposal aims to introduce new syntax from comprehension for monads to comonads.
+Martin is the reviewer. He asks others attendees for their opinion on this.  Everyone had read the SIP.
+
+**Eugene** referred to original proposal and wishes to see a better motivation for this language feature encouraging use of “plain English” to simplify the use of Scala as practice oriented language. He believes that it could be critical how this SIP can be improved. During the recent conference, organized by Facebook, he spoke with typescript guys that are developing idiomic solutions that would benefit typescript and javascript and allow community users to give their inputs.
+Refers to the paper “Denotation” he linked in a proposal, that is not enough for Scala, but a good start.
+
+**Jorge** is getting back discussion on voting on this proposal and he mentioned that Josh insisted on more examples and suggestions on motivation of this SIP.
+
+**Eugene** wanted to add more syntax (map and flatmap), but Martin opposed to that saying that Scala is quite serious program and needs more reason to add any additional syntax to it. **Martin** would like to see more widespread use of common attic constructs and Libraries, and before doing that, he wouldn’t consider any further change. **Sebastian** agrees with Martin and says that he doesn’t really understand Josh’s and Eugene’s proposal. **Iulian** agrees that the proposal is quite complicated and he wonders how it can be useful. He believes that it is an interesting research direction, but that it needs more users feedbacks in aim to be included in the Scala, therefore questioning if the proposal should be numbered in the current form. Seth and Adriaan agree with Martin and Iulian.
+
+**Conclusion** Proposal discarded unanimously. They will send the feedback to the author.
+
+### [SIP-33 - Match infix & prefix types to meet expression rules](http://docs.scala-lang.org/sips/make-types-behave-like-expressions.html)
+
+Skipped since they are waiting for the prototype from the author of the proposal.
+
+### Scala Library Changes
+ [YouTube time 10:52](https://youtu.be/6rKa4OV7GfM?t=652)
+
+ **Adriaan** starts presentation and notes that feedbacks on his proposal are available through the 2.13 platform. It is more reorganization of things in different modules. He suggests list of packages, from the ticket, that he believes shouldn’t be in the core:
+ *Scala concurrent*, *Scala.ref*, *Scala.cis*, *Scala.compat* (that is already totally deprecated), *Scala.text* (that has already couple of things that are deprecated), *Scala.util*; whereas *Scala.io* and *Scala.sys* are good candidates for replacements with better community modules.
```

Comment:
scala.cis? @jvican do you know which one he was referring to?

Reply:
Yeah, `scala.sys` 👍